### ISSUE TITLE
xcode error: libobjc.A.dylib`objc_msgSend

### ISSUE THREAD

**Issue by zenmanenergy** [phonegap/phonegap-plugin-push#84]
I'm not sure what's causing this, but I started getting this error in Xcode when I tried running my app. I have gotten the plugin to work, but now I get this error. It happens right after the "Resetting Plugins due to page load" message in Xcode.

Anyone know what this is? 

libobjc.A.dylib`objc_msgSend:
    0x305cef60 <+0>:  cbz    r0, 0x305cef9e            ; <+62>
    0x305cef62 <+2>:  ldr.w  r9, [r0]

->  0x305cef66 <+6>:  ldrh.w r12, [r9, #0xc]

    0x305cef6a <+10>: ldr.w  r9, [r9, #0x8]
    0x305cef6e <+14>: and.w  r12, r12, r1
    0x305cef72 <+18>: add.w  r9, r9, r12, lsl #3
    0x305cef76 <+22>: ldr.w  r12, [r9]
    0x305cef7a <+26>: teq.w  r12, r1
    0x305cef7e <+30>: bne    0x305cef86                ; <+38>
    0x305cef80 <+32>: ldr.w  r12, [r9, #0x4]
    0x305cef84 <+36>: bx     r12
    0x305cef86 <+38>: cmp.w  r12, #0x1
    0x305cef8a <+42>: blo    0x305cef98                ; <+56>
    0x305cef8c <+44>: it     eq
    0x305cef8e <+46>: ldreq.w r9, [r9, #0x4]
    0x305cef92 <+50>: ldr    r12, [r9, #8]!
    0x305cef96 <+54>: b      0x305cef7a                ; <+26>
    0x305cef98 <+56>: ldr.w  r9, [r0]
    0x305cef9c <+60>: b      0x305cf1e0                ; _objc_msgSend_uncached
    0x305cef9e <+62>: mov.w  r1, #0x0
    0x305cefa2 <+66>: bx     lr
    0x305cefa4 <+68>: nop    
    0x305cefa6 <+70>: nop    
    0x305cefa8 <+72>: nop    
    0x305cefaa <+74>: nop    
    0x305cefac <+76>: nop    
    0x305cefae <+78>: nop    
    0x305cefb0 <+80>: nop    
    0x305cefb2 <+82>: nop    
    0x305cefb4 <+84>: nop    
    0x305cefb6 <+86>: nop    
    0x305cefb8 <+88>: nop    
    0x305cefba <+90>: nop    
    0x305cefbc <+92>: nop    
    0x305cefbe <+94>: nop

**Comment by zenmanenergy** [comment]
I turned on NSZombie because of this stackoverflow suggestion:

http://stackoverflow.com/questions/5386160/how-to-enable-nszombie-in-xcode

It now gives me this error:

2015-08-28 12:30:31.941 Fridge Alive[1224:198565] *** -[UIViewAnimationState _accessibilitySetAnimationsInProgress:]: message sent to deallocated instance 0x1864ca90

Anyone know what that is?

**Comment by macdonst** [comment]
@zenmanenergy I haven't ever seen this. Can you provide a reproduction scenario?

**Comment by zenmanenergy** [comment]
I did quite a bit of googling on this and so far the best solution I had, which sucks, was to remove the IOS platform from cordova, then add it again. Basically rebuild the project from scratch. Now I'm not getting the errors. 

I'm not convinced it had anything to do with the plugin, just something funky with xcode and it just happened to occur after I switched to this plugin. I'm going to close this issue for now.
